Source/CarthageKitTests/XcodeSpec.swift

```diff
@@ -35,7 +33,8 @@ class XcodeSpec: QuickSpec {
 			_ = try? FileManager.default.removeItem(at: targetFolderURL)
 		}
 
-		describe("determingSwiftInformation") {
+		describe("determineSwiftInformation:") {
+			let currentSwiftVersion = swiftVersion.single()?.value
 			let testSwiftFramework = "Quick.framework"
 			let currentDirectory = URL(fileURLWithPath: FileManager.default.currentDirectoryPath, isDirectory: true)
```

Comment:
Is this current directory assumed (or, maybe guaranteed) to be the enclosing directory of `CarthageKitTests.xctest`?

Is `Quick.framework` assumed to be there as a built product of a (not executed by `CarthageKitTests.xctest`) Xcode build?